.github/CONTRIBUTING.md

```diff
@@ -4,6 +4,11 @@ First off, thank you for considering contributing to GNOLL.
 
 Following these guidelines helps to communicate that you respect the time of the developers managing and developing this open source project. In return, they should reciprocate that respect in addressing your issue, assessing changes, and helping you finalize your pull requests.
 
+We're so happy you're considering contributing! Thank you! We hope this document will make it easier to do that.
```

Comment:
We should move this greeting a little higher in the file where we are already thanking the reader